### PR TITLE
Allow user to edit their own request

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: pkgdb2/ui/packages.py [hunk 1]
```diff
@@ -349,9 +351,10 @@ def package_request_edit(package, action_id):
         )
 
     # Check user is the pkg/pkgdb admin
+    pkg_admin = pkgdblib.has_acls(
+        SESSION, flask.g.fas_user.username, package, 'approveacls')
```
Comment (ralphbean) [inline pkgdb2/ui/packages.py]: How will this line work if ``package=None``?

Comment (pypingou) [inline pkgdb2/ui/packages.py]: Local tests showed it behaves correctly but I've added a second check in has_acls itself to cover the situation (and avoid querying the DB for nothing)

**Comment by ralphbean** [comment]
Cool.  :+1:.

**Comment by pypingou** [comment]
Thanks @ralphbean

**Pull request merged**